src/app_main.c

```diff
@@ -467,7 +467,8 @@ void handleApdu(volatile uint32_t *flags, volatile uint32_t *tx, uint32_t rx) {
                     *tx += 65;
 
                     // must be the last bip32 the user "saw" for signing to work.
```

Comment:
Can we remove "`// must be the last bip32 the user "saw" for signing to work.`" too?